Data.R

```diff
@@ -1,97 +1,131 @@
+#'---
+#' title: "[TRIPOD 1] Identify the study as developing and/or validating a multivariable prediction model, the target population, and the outcome to be predicted."
+#' author: 'Author One ^1,✉^, Author Two ^1^'
+#' abstract: |
+#'  | [TRIPOD 2] Provide a summary of objectives, study design, setting, participants, sample size, predictors, outcome, statistical analysis, results, and conclusions.
+#' documentclass: article
+#' description: 'Manuscript'
+#' clean: false
+#' self_contained: true
+#' number_sections: false
+#' keep_md: true
+#' fig_caption: true
+#' css: production.css
+#' output:
+#'   html_document:
+#'    code_folding: hide
+#'    toc: true
+#'    toc_float: true
+#' ---
+#'
 #' Loading R libraries
+#+ init, echo=FALSE, message=FALSE, warning=FALSE
+debug <- 0;
+knitr::opts_chunk$set(echo=debug>-1, warning=debug>0, message=debug>0, max.print=42);
 library(ggplot2)
 library(dplyr)
 library(GGally)
 library(rio)
 library(pander)
 library(maps)
+library(printr)
+options(max.print=42)
+panderOptions('table.split.table',Inf); panderOptions('table.split.cells',Inf);
```

Comment:
`printr` is a cool new library I found (before running this for the first time, do `install.packages('printr')` ). Now you don't have to put `pander()` after every expression that returns a table-- it automatically detects tabular output and formats it. 

The  two `panderOptions()`  settings prevent tables from splitting into multiple segments.

The `options(max.pring=42)` is intended to limit output from any command to 42 entries, so it doesn't fill your screen or report with output (though it's not really effective, so I suggest skipping that line, I should not have left it in).